File: Without Node Module/Xampp/htdocs/core/week-1/c-programs/programs/36.php
<?php
    $array = [5, 2, 8, 1, 3];
    $searchElement = 8;

    function linearSearch($array, $searchElement) {
        foreach ($array as $key => $value) {
            if ($value == $searchElement) {
                return $key;
            }
        }
        return -1;
    }

    $result = linearSearch($array, $searchElement);
    echo "Linear Search: Element $searchElement found at index $result";
?>

File: One Task/Task 1/c programming/36.c
// Linear Search
#include <stdio.h>

int main() {
    int arr[] = {56,78,45,29,45}, n = 45;

    for(int i=0; i<5; i++) {
        if(arr[i] == n) {
            printf("Search Found at position %d\n", (i+1));
        }
    }
    return 0;
}